config.docker.register.yaml

```diff
@@ -33,9 +32,8 @@ externalConfigDirectory: /tmp
 downloadConfigs: true
 
 # can be http: file s3: or classpath: url
-fieldsYamlLocation: classpath://config/fields.yaml
-
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: http://field/records.yaml?page-size=5000
```

Comment:
If the ENVIRONMENT variable were passed in we should be able to allow the app to read direct from field.alpha.openregister.org etc In dropwizard we should be able to use field.${ENVIRONMENT:alpha}.openregister.org . Could we then dispense with docker-compose-basic?